### PR TITLE
Remove .person-card-featured .more-details SCSS rule set

### LINKED ISSUE #2640: Remove .person-card-featured .more-details SCSS rule set
It doesn't seem like we have any `.more-details` elements in `.person-card-featured`. If that's really the case let's remove its SCSS.

### PR DESCRIPTION
**Pull request by HSR05**
Closes #2640 
Related PRs/issues #

## Checklist

**Tests**
- [x] Is the code I'm adding covered by tests?

**Changes in Models:**
- [ ] Did I update or add new fake data?
- [ ] Did I squash my migration?
- [ ] If my changes are not backward-compatible, did I schedule a deploy with the rest of the team?

**Documentation:**
- [x] Is my code documented?
- [ ] Did I update the READMEs or wagtail documentation?

### CODE REVIEW

File: source/js/components/people/people.scss [hunk 1]
```diff
@@ -9,14 +9,12 @@
   }
 
   .person-card-featured {
-    .more-details {
-      background-color: $blue;
-      color: white;
+    
 
       &:hover {
```
Comment (Pomax) [inline source/js/components/people/people.scss]: this has now changed from being `.person-cared-featured .more-details:hover` to `.person-cared-featured:hover`. If nothing uses `.more-details` then its associated pseudo rules should not be kept.

Comment (HSR05) [reply source/js/components/people/people.scss]: My bad,
working on it. 👍

File: source/js/components/people/people.scss [hunk 2]
```diff
@@ -8,16 +8,6 @@
...
-  .person-card-featured {
-    .more-details {
-      background-color: $blue;
-      color: white;
-
-      &:hover {
-        background-color: $dark-blue;
-      }
-    }
-  }
```
Comment (Pomax) [inline source/js/components/people/people.scss]: One more fix required: the SCSS now has two empty lines, which is an error as far as the CSS linter is concerned: when you make changes, and before you push up your commit, remember to run `npm test` to make sure there are no errors left according to the various error checked.

Comment (HSR05) [reply source/js/components/people/people.scss]: Will run npm test from now on, Everything else was good?

Comment (HSR05) [reply source/js/components/people/people.scss]: Submitted the change requested. Thankyou for guiding me through this. Looking forward to contribute more.

Comment (Pomax) [reply source/js/components/people/people.scss]: yep, everything else seemed fine to me.

**Comment by Pomax** [comment]
tagging @mmmavis for approval.

**Comment by mmmavis** [comment]
@Pomax 👀 ?

> Merging is blocked
Merging can be performed automatically once the requested changes are addressed.

**Pull request merged**